man/gemfile.5.ronn

```diff
@@ -69,6 +69,19 @@ the Ruby version that the engine is compatible with.
 Each application _may_ specify a Ruby engine. If an engine is specified, an
 engine version _must_ also be specified.
 
+_What exactly is an Engine?_
+  - A Ruby engine is an implementation of the Ruby language.
+  - For background: the reference or original implementation of the Ruby
```

Comment:
These dot points need to have space separation between them or they will be displayed as a single paragraph